app/views/pages/item-detail/item-attachments.js

```diff
@@ -0,0 +1,278 @@
+import classNames from "classnames";
+
+const CarouselSettings = {
+  dots: false,
+  infinite: true,
+  slidesToShow: 3,
+  speed: 500,
+  slidesToScroll: 1,
+  responsive: [
+    {
+      breakpoint: 1024,
+      settings: {
+        slidesToShow: 3,
+        slidesToScroll: 3,
+        infinite: true
+      }
+    },
+    {
+      breakpoint: 600,
+      settings: {
+        slidesToShow: 2,
+        slidesToScroll: 2
+      }
+    },
+    {
+      breakpoint: 480,
+      settings: {
+        slidesToShow: 1,
+        slidesToScroll: 1
+      }
+    }
+  ]
+};
+
+var ItemAttachments = React.createClass({
+
+  mixins: [State, ItemDetailMixin],
+
+  getInitialState() {
+    return {
+      panelOpen: false
+    }
+  },
+
+  propTypes() {
+    attachments: React.PropTypes.shape({
+      _href: React.PropTypes.string,
+      created_at: React.PropTypes.string,
+      created_by: React.PropsTypes.shape({
+        account: React.PropTypes.object,
+        created_at: React.PropTypes.string,
+        email: React.PropTypes.string,
+        first_name: React.PropTypes.string,
+        id: React.PropTypes.number,
+        last_login: React.PropTypes.string,
+        last_name: React.PropTypes.string
+      }),
+      href: React.PropTypes.string,
+      id: React.PropTypes.number,
+      item: React.PropTypes.object,
+      name: React.PropTypes.string
+    })
+  },
+
+  togglePanel() {
+    this.setState({panelOpen: !this.state.panelOpen});
+  },
+
+  counts(imageCount, fileCount) {
+    let counts = [];
+
+    if (!imageCount && !fileCount) {
+      counts.push(<li key="no-attachment">No Attachments</li>)
+    }
+
+    if (imageCount) {
+      counts.push(<li key="count__images">Images: {imageCount}</li>)
+    }
+    if (fileCount) {
+      counts.push(<li key="count__files">Files: {fileCount}</li>);
+    }
+
+    return counts;
+  },
+
+  attachmentsHeader(imagesCount, filesCount) {
+    let counts = this.counts(imagesCount, filesCount);
+    let headerClasses = classNames({
+      'header-dark': true,
+      'open': this.state.panelOpen
+    });
+    let toggleClasses = classNames({
+      'toggle': true,
+      'transparent': !this.props.attachments.length
+    })
+    let caretClasses = `glyphicon glyphicon-menu-${this.caretState()}`;
+
+    return (
+      <div className={headerClasses}>
+        <a className={toggleClasses} onClick={this.togglePanel}>
+          <span aria-hidden="true" className={caretClasses}/>
+        </a>
+        <div className="sep-vertical"></div>
+        <div className="title">Attachments</div>
+        <ul className="attachments__info-list">{counts}</ul>
+      </div>
+    )
+  },
+
+  attachmentsViewer(images) {
+    if (images && images.length) {
+      return _.map(images, (image, i) => {
+        return (
+          <li key={i}>
+            {this.openNewTabLink(image.href, image.name)}
+          </li>
+        )
+      })
+    }
+    /*
+      TODO: Use when have embedly
+      if (this.props.attachments) {
```

Comment:
We don't need to worry about embedly here. In fact, we can pass `image.href` straight into an image tag for a preview and it will work just fine:

```
<img src={image.href} width="50" />
```

We can keep the behavior of opening it in a new tab, that's 💯 compared to a lightbox / modal